ray-operator/apis/ray/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -6,7 +6,7 @@
 package v1alpha1
 
 import (
-	v1 "k8s.io/api/core/v1"
```

Comment:
em. not sure why it get changed. what's the controller-runtime version you are using?

Reply:
thanks, accidental find/replace, I think. Reverted